client/src/assets/icons/toggle-check.tsx

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
+import '../../components/settings/toggle-setting.css';
+
```

Comment:
You are importing the file twice, one in here and the other in radio or button setting

```suggestion
```

Reply:
Correct, it could be removed, and it works fine.